main.go

```diff
@@ -89,9 +90,10 @@ func main() {
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: metricsAddr + "9443",
```

Comment:
Are you sure about this? I would assume that a bind address would need to look like `host:port`.
```suggestion
			BindAddress: metricsAddr + ":9443",
```

Reply:
yeah, that make sense